wpilibc/Athena/src/PIDController.cpp

```diff
@@ -435,28 +410,28 @@ void PIDController::SetTolerance(float percent) {
 }
 
 /*
- * Set the percentage error which is considered tolerable for use with
+ * Set the absolute error which is considered tolerable for use with
```

Comment:
I switched the order to make them consistent with the order in which they appear in wpilibj and the C++ header.